packages/app-js/src/index.tsx

```diff
@@ -102,15 +135,24 @@ class App extends React.PureComponent<Props, State> {
       window: null
     };
 
-    const exec = `(async ({${Object.keys(injected).join(',')}}) => {
-      try {
-        ${code}
-      } catch (error) {
-        console.error(error);
-      }
-    })(injected);`;
+    // squash into a single line so exceptions (with linenumbers) maps to the same line/origin
+    // as we have in the editor view (TODO: Make the console.error here actually return the full stack)
```

Comment:
Nice, haven't thought of that!

Reply:
This happens when you think of it, squash it into a single line and never comment on it. My bad.